bridge/server/api/index.js

```diff
@@ -30,7 +30,7 @@ module.exports = (params) => {
     try {
       const result = await axios({
         method: req.method,
-        url: `${apiUrl}${req.url}`,
+        url: `${apiUrl}/${req.url}`,
```

Comment:
@ermin-muratovic Is this change correct? (also see line 66)

Reply:
This `/` here is not necessary, because `req.url` starts with a `/` and therefore you would get a double `/` here.
And the `apiUrl` in the server-part actually has not changed. (same on line 66)